src/utils.test.js

```diff
@@ -6,9 +6,65 @@ const {
   encryptValue,
   decryptValue,
   isInvalidValidPassword,
+  hashDappContent,
 } = require('./utils');
 
+const dAppContent = {
+  name: { 'en-us': 'DApp Name', de: 'DApp Name' },
+  engine: '0.1.0',
+  image: 'iVBORw0KGgoAAAANSUhEUgAAAH0AAAB9CAYAAACPg',
+  used_signing_key:
+    '16a6e448c2e1e209ab559232a98e3d0c25030e05158c22710c207fec1f2e4fbf',
+  code: 'get:function(){return"Text"}',
+};
+
 describe('utils', () => {
+  test('hashDappContent', () => {
```

Comment:
this test makes little sense to me of course the returned result will be different if you pass in not the same params. The right way to test this is, to compute the expected hash manually. when you have the manually computed hash you use `hashDAppContent` and compare those results

Reply:
done